test/tasks/example/sep/008-opcm-upgrade-v400/config.toml

```diff
@@ -0,0 +1,15 @@
+templateName = "OPCMUpgradeV400"
+
+[[l2chains]]
+chainId = 11155420
+name = "OP Sepolia Testnet"
+
+[[opcmUpgrades]]
+chainId = 11155420
+absolutePrestate="0x0399cfb018011977a027d1e7a85eb7ff101aec9bfc7d6500abac944c47a4581f"
+expectedValidationErrors = ""
+
+[addresses]
+OPCM = "0x355fd2a98fc11f5e4a91d9aa4c0041d252a77583" # Freshly deployed on Sepolia
+StandardValidatorV400 = "0xd0233aa63b11c2b93f68e306a16f2786fb11d0d1" # Freshly deployed on Sepolia
+EthLockboxProxy = "0x40B7b580090D22727aed809B3845559aDcf15711" # Freshly deployed on Sepolia
```

Comment:
1. This `EthLockboxProxy` address doesn't actually [exist on Sepolia](https://sepolia.etherscan.io/address/0x40B7b580090D22727aed809B3845559aDcf15711), so it must be an address which is deployed during the upgrade. Since we're doing deterministic deployments that's fine, but the comment is a bit confusing.  
2. There will actually be one new `EthLockboxProxy` per chainId, so this value needs to go into `[[opcmUpgrades]]`.